lib/jekyll-readme-index/generator.rb

```diff
@@ -26,15 +28,38 @@ def readme
     end
 
     def index?
-      (site.pages + site.static_files).any? { |file| file.url =~ INDEX_REGEX }
+      relative_index?(readme)
     end
 
     def page
+      relative_page(readme)
```

Comment:
it is the `readme` method not object. this is just maintaining the previous page/readme/index? interface in a DRY way